src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography, Grid } from '@mui/material';
+import { StyledFooter } from './StyledFooter';
+import { theme } from '../../Theme/Theme';
+import NavigationList from '../NavigationList/NavigationList';
+const categories = { listMainText: 'Shope By Category', list: ['HandBags', 'Watches', 'Skincare', 'Jewllery', 'Apparels', 'HandBags', 'Watches'] };
+const products = { listMainText: 'Shope By Product', list: ['HandBags', 'Watches', 'Skincare'] };
+import LocationOnOutlinedIcon from '@mui/icons-material/LocationOnOutlined';
+import FacebookOutlinedIcon from '@mui/icons-material/FacebookOutlined';
+import TextWithIcon from '../TextWithIcon/TextWithIcon';
+import SocialMediaLinks from '../SocialMediaLinks/SocialMediaLinks';
+
+const socialMedia = [{ Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }, { Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }, { Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }, { Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }]
+
+Footer.propTypes = {
+    categories: PropTypes.object.isRequired,
+    products: PropTypes.object.isRequired,
+    location: PropTypes.string.isRequired,
+    rights: PropTypes.string.isRequired,
+    socialMedia: PropTypes.array.isRequired,
+}
+
+Footer.defaultProps = {
+    categories: categories,
+    products: products,
+    location: 'United Sates',
+    rights: '© 2021 | Cora Leviene All Rights Reserved',
+    socialMedia: socialMedia
+};
+
```

Comment:
It is more common and clear to place propTypes and defaultProps after the component definition but before the export statement.